File: POOJAVA/src/clasesPOO/Cursos.java
package clasesPOO;

public class Cursos 
{
    String idCurso;
    String NCurso;
    int Creditos;

    public Cursos(String idCu,String NCu,int cred)
    {
        establecerDatos(idCu,NCu,cred);
    }
    public String ObteneridCurso()
    {
       return idCurso;
    }
    public String ObtenerNCurso()
    {
       return NCurso;
    }
    public int ObtenerCreditos()
    {
       return Creditos;
    }
    void  establecerDatos(String idCur,String NCur,int credi)
    {
        idCurso=idCur;
        NCurso=NCur;
        Creditos=credi;
    }
}

File: POOPHP/app/Model/Cursos.php
<?php
namespace App\Model;
class Cursos
{
    public $idCurso;
    public $NCurso;
    public $Creditos;

   public function __construct(String $idCu,String $NCu,int $cred)
    {
        $this->establecerDatos($idCu,$NCu,$cred);
    }
    public function ObteneridCurso()
    {
        return $this->idCurso;
    }
    public function ObtenerNCurso()
    {
        return $this->NCurso;
    }
    public function ObtenerCreditos()
    {
        return $this->Creditos;
    }
    public function establecerDatos(String $idCur,String $NCur,int $credi)
    {
        $this->idCurso=$idCur;
        $this->NCurso=$NCur;
        $this->Creditos=$credi;
    }
}
?>